src/api-reference/direct-connects/hotel-service-4/v4.supported-ota-codes.md

```diff
@@ -0,0 +1,351 @@
+---
```

Comment:
Hi @kunalshah01 . We cannot lift information from another website. If you would like to place a link in the Endpoint or Getting Started docs, that is acceptable but we cannot duplicate this information.